lib/rules/capitalized-comments.js

```diff
@@ -232,7 +232,7 @@ module.exports = {
             const commentWordCharsOnly = commentWithoutAsterisks
                 .replace(WHITESPACE, "");
 
-            const firstWordChar = commentWordCharsOnly[0];
+            const firstWordChar = commentWordCharsOnly[0] || "";
```

Comment:
I think it would be better to put an explicit `length` check above this line, rather than defaulting to an empty string. In all other cases `firstWordChar` will be exactly one character long. Setting it to an empty string in this case might cause confusion/unexpected behavior if the code below this line is refactored and someone forgets to account for the empty-string case.